standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestObjectStore.java

```diff
@@ -439,14 +439,14 @@ public void testPartitionOps() throws Exception {
     Assert.assertEquals(2, numPartitions);
 
     try (AutoCloseable c = deadline()) {
-      objectStore.dropPartition(DEFAULT_CATALOG_NAME, DB1, TABLE1, value1);
```

Comment:
Ideally, I would like to keep the old tests and add new tests that takes in "partName" as argument.